src/locales/en.json

```diff
@@ -93,6 +93,12 @@
     }
   },
   "games": {
+    "quizgame":{
+      "screenshot":{
+        "title":"Quiz Game",
+        "src":"https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7"
+      }
+    },
```

Comment:
"quizgame":{
      "title":"Quiz Game",
      "screenshot": "https://firebasestorage.googleapis.com/v0/b/capstone-istanbul-juniper.appspot.com/o/QuizGame%2Fscreenshot.png?alt=media&token=d2736b79-3595-483a-be77-1afce5ddd4a7",
      ...
    }

I was referring to something like this

Reply:
PTAL

Reply:
nice